Snakefile

```diff
@@ -70,17 +72,30 @@ rule download:
 rule decompress:
     message: "Parsing fasta into sequences and metadata"
     input:
-        sequences = rules.download.output.sequences,
-        metadata = rules.download.output.metadata
+        sequences = "data/sequences_{serotype}.fasta.zst",
+        metadata = "data/metadata_{serotype}.tsv.zst"
     output:
-        sequences = "results/sequences_{serotype}.fasta",
-        metadata = "results/metadata_{serotype}.tsv"
+        sequences = "data/sequences_{serotype}.fasta",
+        metadata = "data/metadata_{serotype}.tsv"
     shell:
         """
         zstd -d -c {input.sequences} > {output.sequences}
         zstd -d -c {input.metadata} > {output.metadata}
         """
 
+rule wrangle_metadata:
```

Comment:
Conceptually, why are we creating a new strain column here (using accession), and then (effectively) swapping it back in the auspice JSON?

I'm guessing it's due to duplicates in the strain names (e.g. denv2 has 4085 rows, with 18 strain names occurring twice, and one strain name occurring 4 times). However should this de-duplication not occur in the ingest phase?